### PR TITLE
Fix for issue 317

### PR DESCRIPTION
**Pull request by suratdas**
Fixes https://github.com/Visual-Regression-Tracker/Visual-Regression-Tracker/issues/317

### CODE REVIEW

File: src/components/TestDetailsDialog/TestDetailsModal.tsx [hunk 1]
```diff
@@ -359,7 +365,7 @@ const TestDetailsModal: React.FunctionComponent<{
                 <Grid item>
                   <IconButton
                     disabled={!selectedRectId || ignoreAreas.length === 0}
-                    onClick={() => applyIgnoreArea()}
+                    onClick={() => setApplyIgnoreDialogOpen(!applyIgnoreDialogOpen)}
```
Comment (pashidlos) [inline src/components/TestDetailsDialog/TestDetailsModal.tsx:368]: could reuse `toggleApplyIgnoreDialogOpen`?

File: src/components/TestDetailsDialog/TestDetailsModal.tsx [hunk 2]
```diff
@@ -481,6 +487,22 @@ const TestDetailsModal: React.FunctionComponent<{
         onOriginalSizeClick={setOriginalSize}
         onFitIntoScreenClick={fitStageToScreen}
       />
+      <BaseModal
+        open={applyIgnoreDialogOpen}
+        title={''}
```
Comment (pashidlos) [inline src/components/TestDetailsDialog/TestDetailsModal.tsx:492]: you could reuse text from tooltip 
line 362`Apply selected ignore area to all images in this build.`

File: src/components/TestDetailsDialog/TestDetailsModal.tsx [hunk 3]
```diff
@@ -481,6 +487,22 @@ const TestDetailsModal: React.FunctionComponent<{
...
+        submitButtonText={'Yes'}
+        onCancel={toggleApplyIgnoreDialogOpen}
+        content={
+          <Typography>
+            {`Are you sure you want to apply ignore area to all items in this build?`}
```
Comment (pashidlos) [inline src/components/TestDetailsDialog/TestDetailsModal.tsx:497]: drop a note that all images gonna be re-compared with new ignore area taken into account

**Comment by suratdas** [comment]
Fixed review comments.

**Pull request merged**